### PR TITLE
Exclude paths using the exclude-path option

### PR DESCRIPTION
**Pull request by Cydonia7**

### CODE REVIEW

File: GithubComment/Command/CommentCommand.php [hunk 1]
```diff
@@ -32,18 +32,31 @@ public function configure()
...
+                    'log',
+                    '--format=%s',
+                    '--merges',
+                    sprintf('%s..%s', $input->getArgument('previous-revision'), $input->getArgument('new-revision')),
+                    '--',
+                    '.',
+                ),
+                $excludedPaths
+            )
+        );
```
Comment (lyrixx) [inline GithubComment/Command/CommentCommand.php]: Let's use a more compact version
```
$defaultArgs = [
    'log',
    '--format=%s',
    '--merges',
    sprintf('%s..%s', $input->getArgument('previous-revision'), $input->getArgument('new-revision')),
    '--',
    '.',
]
$cmd = $this->createGitCommand($input, array_merge($defaultArgs, $excludedPaths));
```

File: GithubComment/Command/CommentCommand.php [hunk 2]
```diff
@@ -32,18 +32,31 @@ public function configure()
...
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $cmd = $this->createGitCommand($input, array(
-            'log',
-            '--format=%s',
-            '--merges',
-            sprintf('%s..%s', $input->getArgument('previous-revision'), $input->getArgument('new-revision'))
-        ));
+        $excludedPaths = array_map(function ($path) {
+            return ':!' . $path;
```
Comment (lyrixx) [inline GithubComment/Command/CommentCommand.php]: cs looks wrong

**Comment by Cydonia7** [comment]
Issues addressed.

**Comment by lyrixx** [comment]
Thanks for your work on this new feature!

**Comment by lyrixx** [comment]
I reverted your PR as It does not work.

**Pull request closed without merging**